scripts/build/includeify

```diff
@@ -1,39 +1,38 @@
 #!/usr/bin/env python3
 
 import argparse
-import os
 import pathlib
+import time
 from typing import TYPE_CHECKING, cast
 
 if TYPE_CHECKING:
-    from typing import Protocol, Optional
+    from typing import Protocol
 
     class Arguments(Protocol):
-        name: Optional[str]
+        name: str
         input: pathlib.Path
         output: pathlib.Path
 
 
 parser = argparse.ArgumentParser(
     description="Convert a file into a C-style header file similar to xxd"
 )
-parser.add_argument("-n", "--name", help="Variable name")
+parser.add_argument("-n", "--name", required=True, help="Variable name")
 parser.add_argument("input", type=pathlib.Path, help="Input file")
 parser.add_argument("output", type=pathlib.Path, help="Output file")
 
 args = cast("Arguments", parser.parse_args())
 
-IDENT_FILE_NAME = str(args.input).replace(os.sep, "_").replace(".", "_")
-VARIABLE_NAME = args.name if args.name else IDENT_FILE_NAME
+INCLUDE_GUARD = f"{args.name.upper()}_{int(time.time())}_H"
```

Comment:
What's the resolution of time.time() ?

Reply:
Unix time. I could also use a random number